test/irb/test_init.rb

```diff
@@ -35,132 +40,82 @@ def test_setup_with_minimum_argv_does_not_change_dollar0
     end
 
     def test_rc_file
-      verbose, $VERBOSE = $VERBOSE, nil
       tmpdir = @tmpdir
       Dir.chdir(tmpdir) do
         ENV["XDG_CONFIG_HOME"] = "#{tmpdir}/xdg"
-        IRB.conf[:RC_NAME_GENERATOR] = nil
-        assert_equal(tmpdir+"/.irbrc", IRB.rc_file)
-        assert_equal(tmpdir+"/.irb_history", IRB.rc_file("_history"))
+        reset_rc_name_generators
+        assert_empty(IRB.irbrc_files)
+        assert_equal("#{ENV["HOME"]}/.irb_history", IRB.rc_file("_history"))
```

Comment:
Isn't this the same as `tmpdir`? I understand that it's to emphasise that this value should be home, but I feel it could be misleading if we use `ENV["HOME"]` and `tmpdir` for the same value.

Reply:
I'll remove this `test_rc_file` because history file is tested more in another test `test_rc_files`